### PR TITLE
RPE-670#5 case null check

### PR DESCRIPTION
**Pull request by KevinJCross**
**Before creating a pull request make sure that:**
### JIRA link (if applicable) ###
RPE-670

### Change description ###
adding null check for the case data
Also combining the validations into one class to extract at a later date.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.List;
+
+public interface CallbackProcessor {
```
Comment (timja) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java]: why do you need this interface?

Comment (KevinJCross) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java]: Well not really ... its one of those springy things about object contracts.
Could remove it if ya want

Comment (timja) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java]: would prefer, seems unneeded to me

Comment (KevinJCross) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java]: It seems I got then to  branches mixed in this PR ... just give me a sec

**Pull request merged**